terminal-view/src/main/java/com/termux/view/TerminalView.java

```diff
@@ -764,25 +764,22 @@ public void updateSize() {
 
     @Override
     protected void onDraw(Canvas canvas) {
-        if (mEmulator == null) {
-            canvas.drawColor(0XFF000000);
-        } else {
-            mRenderer.render(mEmulator, canvas, mTopRow, mSelY1, mSelY2, mSelX1, mSelX2);
+        canvas.drawColor(0XFF000000);
+        mRenderer.render(mEmulator, canvas, mTopRow, mSelY1, mSelY2, mSelX1, mSelX2);
```

Comment:
You should still protect the `mRenderer.render()` call and check for `mEmulator == null`.